src/CMakeLists.txt

```diff
@@ -0,0 +1,116 @@
+if(VMA_BUILD_EXAMPLE_APP_SHADERS)
+    add_subdirectory(Shaders)
+endif()
+
+if(VMA_BUILD_REPLAY_APP)
+    add_subdirectory(VmaReplay)
+endif()
+
+set(VMA_LIBRARY_SOURCE_FILES
+    VmaUsage.cpp
+)
+
+# Use folder structure as filters in project map
+foreach(FILE ${VMA_LIBRARY_SOURCE_FILES})
```

Comment:
Is this grouping into source and header files really needed, or just a purely visual grouping for Visual Studio? If the latter, maybe we could drop this, as there are only few files in each project, while this would make CMakeLists.txt files shorter by a large %.

Reply:
It's not needed. I will remove it.

Reply:
Done.